Web_Development/Javascript/7.The DOM/DOM - Examples/Example3 - Tab switching/README.md

```diff
@@ -0,0 +1,3 @@
+# Example3 - Tab Switching
+
+<img src='example3.gif'>
```

Comment:
Use Code blocks.

Reply:
Required changes made in the latest commit.